main.tf

```diff
@@ -1,14 +1,18 @@
 locals {
   node_name_suffix = "${var.cluster_id}.${var.base_domain}"
+  create_privnet_subnet = var.subnet_uuid == "" ? 1 : 0
```

Comment:
This will be a breaking change for existing clusters, as the name in the Terraform state for the subnet managed by the module will change.